xml/System.Windows.Forms/MdiClient.xml

```diff
@@ -176,8 +169,8 @@
         <ReturnType>System.Windows.Forms.CreateParams</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the required creation parameters when the control handle is created.</summary>
+        <value>The required creation parameters when the handle to the control is created.</value>
```

Comment:
I think `control handle` is more common.

Reply:
@Tanya-Solyanik  Not sure if this is an actionable comment. Should the documentation be updated? Feel free to provide a suggestion if you think the description can be improved.

Reply:
```suggestion
        <value>The required creation parameters when the control handle is created.</value>
```